src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -72,6 +72,7 @@ function AnchorRenderer({tnode, key, style}) {
     // An auth token is needed to download Expensify chat attachments
     const isAttachment = Boolean(htmlAttribs['data-expensify-source']);
     const fileName = lodashGet(tnode, 'domNode.children[0].data', '');
+    const parentStyle = lodashGet(tnode, 'parent.styles.nativeTextRet', {});
```

Comment:
This code looks OK to me, but seems potentially brittle because it's relying on some internal detail of the html library we are using. 

e.g. I'm not really sure what `parent.styles.nativeTextRet` is or whether we can depend on it to always exist? I also don't have an alternative suggestion so I'm fine with this for now - but would have preferred more comments in the code to add context about why we are doing this.